### DISCUSSION TITLE
INS keybinding lost?

### DISCUSSION THREAD

**Discussion by darquan0** [pulsar-edit/.github#298] [General]
Somehow, somewhy, INS key does nothing.
I'm not really sure, but I think sometime before pulsar had switch between insert and override mode...
Didn't change anything in config. Don't know when it actually happened.
Tried to look in keybindings, but it has nothing with insert except copy/paste actions on ctrl+ins, etc.

Searched theese discussions, but nothing relevant found.

Please advise...

**Comment by asiloisad** [comment]
Overtype mode isn't built in. Check community packages:
* https://web.pulsar-edit.dev/packages/overtype-mode
* https://web.pulsar-edit.dev/packages/atom-overtype-mode
